internal/cmd/controller/imagescan/tagscan_job_test.go

```diff
@@ -0,0 +1,58 @@
+package imagescan
+
+import (
+	"testing"
+
+	fleet "github.com/rancher/fleet/pkg/apis/fleet.cattle.io/v1alpha1"
+)
+
+var alphabeticalVersions = []string{"a", "b", "c"}
+
+func TestLatestTag(t *testing.T) {
+	tests := []struct{
+		name, want string
+		policy fleet.ImagePolicyChoice
+	}{
+		{
+			name: "alphabetical asc lowercase",
+			policy: fleet.ImagePolicyChoice{
+				Alphabetical: &fleet.AlphabeticalPolicy{Order: "asc"},
+			},
+			want: "a",
+		}, 
+		{
+			name: "alphabetical asc uppercase",
+			policy: fleet.ImagePolicyChoice{
+				Alphabetical: &fleet.AlphabeticalPolicy{Order: "ASC"},
+			},
+			want: "a",
+		}, 
+		{
+			name: "alphabetical desc lowercase",
+			policy: fleet.ImagePolicyChoice{
+				Alphabetical: &fleet.AlphabeticalPolicy{Order: "desc"},
+			},
+			want: "c",
+		}, 
+		{
+			name: "alphabetical desc uppercase",
+			policy: fleet.ImagePolicyChoice{
+				Alphabetical: &fleet.AlphabeticalPolicy{Order: "DESC"},
+			},
+			want: "c",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := latestTag(tt.policy, alphabeticalVersions)
+			if err != nil {
+				t.Errorf("Error calling latestTag: %v", err)
```

Comment:
This isn't expected to return an error in these tests, so the next comparison isn't really valid?

I think this should be `t.Fatal()` see https://go.dev/wiki/TestComments#keep-going

> If you’re using subtests (and you’re inside a call to t.Run), then t.Fatal ends the current subtest and allows your test case to progress to the next subtest, so use t.Fatal.

Reply:
Thank you for the review. I am still new to Go, so I may make some mistakes. 

What do you mean by the next comparison isn't really valid? Does that mean checking for the `err` from `latestTag` is not necessary?

Reply:
I see. Thank you for the correction.